runtime/src/main/java/org/corfudb/protocols/wireprotocol/orchestrator/OrchestratorRequestType.java

```diff
@@ -0,0 +1,52 @@
+package org.corfudb.protocols.wireprotocol.orchestrator;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+/**
+ * The type of requests that can be made to the Orchestrator Service.
+ * 
+ * @author Maithem
+ */
+@AllArgsConstructor
+public enum OrchestratorRequestType {
+
+    /**
+     * Query a workflow id
+     */
+    QUERY(0),
+
+    /**
+     * Add a new node to the cluster
+     */
+    ADD_NODE(1);
+
+    @Getter
+    public int type;
+
+    /**
+     * Create a workflow instance from an orchestrator request
+     *
+     * @param request Orchestrator request
+     * @return Workflow instance
+     */
+    public Workflow getWorkflow(OrchestratorRequest request) {
+        Request payload = request.getRequest();
+        switch (payload.getType()) {
+            case ADD_NODE: return new AddNodeWorkflow(payload);
+            default:
+                throw new IllegalArgumentException("Unknown request type " + payload.getType());
+        }
+    }
+
+    /**
+     * Map an int to an enum.
+     */
+    public static OrchestratorRequestType forNumber(int value) {
```

Comment:
Please generate this in a static way. We don't want to maintain this table.

For example, see typeMap in LogEntry.